content/scout/quickstart.md

```diff
@@ -19,7 +16,11 @@ This guide takes a vulnerable container image and shows you how to use Docker
 Scout to identify and fix the vulnerabilities, compare image versions over time,
 and share the results with your team.
 
-## Setup
+_The following video shows an end-to-end workflow of using Docker Scout to remediate a reported vulnerability_.
```

Comment:
Why is the sentence-ending period not italicized?

Reply:
Arguably none of this should be italicized, but this was a lazy copy/paste on my part so I blame whoever came before me 🙈